DatabaseBuddy/ViewModel/MainWindowViewModel.cs

```diff
@@ -2004,6 +2110,9 @@ private void __InitializeCommands()
             ChangeBaseTheme = new DelegateCommand<object>(Execute_ChangeBaseTheme);
             RestartSQLServerInstance = new DelegateCommand<object>(Execute_RestartService);
             RestartAsAdmin = new DelegateCommand<object>(Execute_RestartAsAdmin);
+            IncreaseScaling = new DelegateCommand<object>(Execute_IncreaseScaling);
```

Comment:
In my opinion running this every time you rescale is way to much

Reply:
Would prefer every time the program is startet, the reload button is pressed or the connection is reestablished